tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/effects/debugger_effects.ts

```diff
@@ -105,6 +113,68 @@ import {
 /** @typehack */ import * as _typeHackNgrxStore from '@ngrx/store/src/models';
 /** @typehack */ import * as _typeHackNgrxEffects from '@ngrx/effects/effects';
 
+const DEFAULT_MINIMUM_POLLING_INTERVAL = 2 * 1e3;
+const DEFAULT_MAXIMUM_POLLING_INTERVAL = 60 * 1e3;
+const DEFAULT_POLLING_BACKOFF_FACTOR = 2;
+
+// Minimum polling interval in milliseconds.
+let minimumPollingInterval = DEFAULT_MINIMUM_POLLING_INTERVAL;
+// Maximum polling interval in miliseconds.
+let maximumPollingInterval = DEFAULT_MAXIMUM_POLLING_INTERVAL;
+// Backoff behavior takes effect when time since the elapsed time between the
+// most recent arrival of new polling result and the most recent polling event
+// exceeds `pollingBackoffFactor * minimumPollingInterval`.
+let pollingBackoffFactor = DEFAULT_POLLING_BACKOFF_FACTOR;
```

Comment:
1. Please do not use have stateful module
2. These are probably unused after removing the deadcode?